app/views/ExposureHistory/ExposureHistory.js

```diff
@@ -0,0 +1,136 @@
+import { css } from '@emotion/native';
+import dayjs from 'dayjs';
+import React, { useEffect, useState } from 'react';
+import { BackHandler, ScrollView } from 'react-native';
+
+import NavigationBarWrapper from '../../components/NavigationBarWrapper';
+import { Typography } from '../../components/Typography';
+import { BIN_DURATION, MAX_EXPOSURE_WINDOW } from '../../constants/history';
+import { CROSSED_PATHS } from '../../constants/storage';
+import { Theme, charcoal, defaultTheme } from '../../constants/themes';
+import { GetStoreData } from '../../helpers/General';
+import languages from '../../locales/languages';
+import { DetailedHistory } from './DetailedHistory';
+
+const NO_HISTORY = [];
+
+export const ExposureHistoryScreen = ({ navigation }) => {
+  const [history, setHistory] = useState(null);
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    async function fetchData() {
+      let dayBins = await GetStoreData(CROSSED_PATHS);
+
+      // dayBins = generateFakeIntersections(6); // handy for creating faux data
+
```

Comment:
I'm guessing some fixup in logic right about here.  I was able to do this in both Android and iOS by setting locations (so at least 1 location was set), picking the test authority, and then reloading the app.  The crossed results array looks like this 
`Crossing results:  [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]`

![Screen Shot 2020-04-13 at 5 54 26 PM](https://user-images.githubusercontent.com/4031514/79168978-cc170c00-7db0-11ea-88a3-383f9f5d8b80.png)

Reply:
It only shows 14 days despite what's in dayBins

I've updated it to only show Possible exposure (charcoal bg) if there was an actual hit

And updated it to show orange on current day circle if there's an exposure